packages/modules/delete-itwin/src/components/DeleteITwin.test.tsx

```diff
@@ -7,6 +7,18 @@ import React from "react";
 
 import { DeleteITwin } from "./DeleteITwin";
 
+jest.mock("@itwin/itwinui-react", () => {
```

Comment:
What do you think about spying instead to avoid the use of requireActual?

Something like: 

```tsx
import * as iTwinUiReact from "@itwin/itwinui-react"
jest.spyOn(iTwinUiReact, "useToaster").mockImplementation(() => undefined)
```

Reply:
I don't think requireActual is bad. I've seen it used across multiple products in Bentley. Secondly with spy we need to return every single type which exists in the toaster which we don't need since the component doesn't rely on all types.

Reply:
> Secondly with spy we need to return every single type which exists in the toaster 

I don't think this is the case? The `mockImplementation` or `mockReturnValue` would be similar to what you're doing now with `mockReturnValue` off the mock.

But it's fine, wanted to get your thoughts.